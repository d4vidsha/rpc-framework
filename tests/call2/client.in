init ::1 6000
find add2
find echo2
call echo2 echo2
0 9
COMP30023
call add2 add2
80 -40
close
