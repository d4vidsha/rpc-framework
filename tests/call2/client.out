rpc_init_client: instance 0, addr ::1, port 6000
rpc_find: instance 0, add2
rpc_find: instance 0, returned handle for function add2
rpc_find: instance 0, echo2
rpc_find: instance 0, returned handle for function echo2
rpc_call: instance 0, calling echo2, data1 = 0, data2 sha256 = 55a10ed...
rpc_call: instance 0, call of echo2 received data1 = 0, data2 sha256 = 55a10ed
rpc_call: instance 0, calling add2, with arguments 80 -40...
rpc_call: instance 0, call of add2 received result 40
rpc_close_client: instance 0
